web/styles/input_pill.css

```diff
@@ -162,7 +162,8 @@
     }
 }
 
-.panel_subscriber_member_list > .pill-container {
+.panel_subscriber_member_list > .pill-container,
+.stream_creator_details > .pill-container {
```

Comment:
It's probably worth doing a prep commit that adds an explicit class name describing what's special about the pills that get this block, and apply that class, rather than just having a list of specific widget class names.

Reply:
Added a new css class `.display_only_pill` for these pills. 

<details><summary>git grep on new css class</summary>
<p>

```css
(zulip-py3-venv) vagrant@fbc57b970081:/srv/zulip$ git grep ".display_only_pill"
web/styles/input_pill.css:.panel_subscriber_member_list > .display_only_pill,
web/styles/input_pill.css:.stream_creator_details > .display_only_pill {
web/templates/stream_settings/new_stream_user.hbs:        {{> ../user_display_only_pill display_value=full_name}}
web/templates/stream_settings/stream_member_list_entry.hbs:        {{> ../user_display_only_pill display_value=name}}
web/templates/stream_settings/stream_settings.hbs:                            {{> ../user_display_only_pill
web/templates/user_display_only_pill.hbs:<span class="pill-container display_only_pill {{#if is_inline}}pill-container-btn inline_with_text_pill{{/if}}">
```

</p>
</details>

Reply:
This likely deserves further cleanup but we can do that in a follow-up PR.